services/skus/controllers.go

```diff
@@ -1033,6 +1033,11 @@ func HandleStripeWebhook(service *Service) handlers.AppHandler {
 					sublogger.Error().Err(err).Msg("failed to update order metadata")
 					return handlers.WrapError(err, "error updating order metadata", http.StatusInternalServerError)
 				}
+				// set paymentProcessor as stripe
+				err = service.Datastore.UpdateOrderMetadata(orderID, paymentProcessor, StripePaymentMethod)
+				if err != nil {
+					return handlers.WrapError(err, "failed to update order to add the payment processor", http.StatusInternalServerError)
```

Comment:
the 500 will trigger the sentry alert but I dont think it logs in AWS. Is it worth logging so we can trace the request

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/1666/commits/bd40563051a0bbc3c1a676a19b7e1e879768fcfa